src/views/index.js

```diff
@@ -0,0 +1,35 @@
+function LoginPage(options={}) {
+    const divContent = document.querySelector('#app');
+    const addPage = function addPage(add) {
+        return `
+        
+        `;
+    }
+    divContent.addEventListener('load', addPage)
+    return ``;
+}
+
+const evaluationLink = document.querySelector('#evaluations');
+const newEvaluationsLink = document.querySelector('#newEvaluation');
+evaluationLink.preventDefault();
+newEvaluationsLink.preventDefault();
+
+const listenerPage1 = function listener() {
```

Comment:
constant and variable names should be a bit more representative. For example evaluationPageListener, evaluationsListPageListener